### ISSUE TITLE
[Adaptive Cards - Expense Report]: While navigating the Expense Report card, Talkback is announcing irrelevant information when activating the 'Show History' button.'

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#374]
### Target Platforms

Android

### SDK Version

1.0.1 (2)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 14
Device: S24
Screen Reader: Talkback
AdaptiveCard Android App version: 1.0.1(2)

**Repro Steps:**

1. Launch the Adaptive cards app on Android.
2. Turn on Talkback.
3. Navigate to 'Import' control and select it.
4. Navigate to 'Sample cards' and select it.
5. Navigate to available dropdown under 'Sample cards'.
6. Navigate to available dropdown under 'Sample cards' and select it.
7. Navigate all the controls under dropdown and select 'Expense Report'.
8. Navigate to 'Show History' button and double tap on it to activate it and Observe the issue.

**Actual Result:**
Talkback is announcing irrelevant information when user is activating 'Show History' button, Here, it is announcing as **'Card content 4 expanded show history collapsed hide history expanded.'**

**Expected Result:**
Talkback should not announce the irrelevant information while activating the 'Show History' button. After activating it. Talkback should announce something like **'Hide history button.'**

**Note:**
Same issue is repro throughout the screen for the similar control.

**User Impact:**
When TalkBack announces irrelevant or unrelated information, it confuses users who rely on auditory cues to understand what just happened. They may not know whether the button worked, what action was triggered, or what content is now available.

**WCAG Reference:**
**https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence**

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/de9d17cd-aa13-4c66-a398-7e6f772a0c9d

![Image](https://github.com/user-attachments/assets/993e87dc-5c18-46ca-af6d-4f21eab5deb1)

### Card JSON

```JSON
NA
```
